### PR TITLE
Bor simulation test

### PR DESCRIPTION
**Pull request by ptsayli**

### CODE REVIEW

File: bor/genesis_test.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+func (suite *GenesisTestSuite) TestInitExportGenesis() {
+	t, app, ctx := suite.T(), suite.app, suite.ctx
+	spancount := 5
+	params := types.DefaultParams()
+	spans := []*hmTypes.Span{}
+	chainID := "15001"
+	start := uint64(0)
+	end := uint64(0)
+	chSim.LoadValidatorSet(4, t, app.StakingKeeper, ctx, false, 10)
+	app.BorKeeper.SetParams(ctx, types.DefaultParams())
```
Comment (tokamak-git) [inline bor/genesis_test.go:42]: `params` holds the same values correct, any reason not to reuse?

File: bor/genesis_test.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	valSet := app.StakingKeeper.GetValidatorSet(ctx)
```
Comment (tokamak-git) [inline bor/genesis_test.go:43]: `LoadValidatorSet` above returns the validators set, any reason we cant use that directly?

File: bor/genesis_test.go [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+		)
+		spans = append(spans, &span)
+	}
+	genesisState := types.NewGenesisState(
+		params,
+		spans,
+	)
+	bor.InitGenesis(ctx, app.BorKeeper, genesisState)
+	actualParams := bor.ExportGenesis(ctx, app.BorKeeper)
+	require.NotNil(t, actualParams)
```
Comment (tokamak-git) [inline bor/genesis_test.go:65]: dont we want to compare the value being returned?

File: bor/genesis_test.go (hunk 1 above)
Comment (ptsayli) [reply bor/genesis_test.go:42]: my bad : 🤗

File: bor/genesis_test.go (hunk 3 above)
Comment (ptsayli) [reply bor/genesis_test.go:65]: can, but imported and exported genesis will not be exact same
so let's keep this as it is

**Pull request merged**